src/app/machines/views/MachineDetails/MachineDetails.test.tsx

```diff
@@ -45,85 +45,67 @@ describe("MachineDetails", () => {
   [
     {
       component: "MachineSummary",
-      route: urls.machines.machine.summary(null),
       path: urls.machines.machine.summary({ id: "abc123" }),
     },
     {
       component: "MachineInstances",
-      route: urls.machines.machine.instances(null),
       path: urls.machines.machine.instances({ id: "abc123" }),
     },
     {
       component: "MachineNetwork",
-      route: urls.machines.machine.network(null),
       path: urls.machines.machine.network({ id: "abc123" }),
     },
     {
       component: "MachineStorage",
-      route: urls.machines.machine.storage(null),
       path: urls.machines.machine.storage({ id: "abc123" }),
     },
     {
       component: "MachinePCIDevices",
-      route: urls.machines.machine.pciDevices(null),
       path: urls.machines.machine.pciDevices({ id: "abc123" }),
     },
     {
       component: "MachineUSBDevices",
-      route: urls.machines.machine.usbDevices(null),
       path: urls.machines.machine.usbDevices({ id: "abc123" }),
     },
     {
       component: "MachineCommissioning",
-      route: urls.machines.machine.commissioning.index(null),
       path: urls.machines.machine.commissioning.index({ id: "abc123" }),
     },
     {
       component: "NodeTestDetails",
-      route: urls.machines.machine.commissioning.scriptResult(null),
       path: urls.machines.machine.commissioning.scriptResult({
         id: "abc123",
         scriptResultId: 1,
       }),
     },
     {
       component: "MachineTests",
-      route: urls.machines.machine.testing.index(null),
       path: urls.machines.machine.testing.index({ id: "abc123" }),
     },
     {
       component: "NodeTestDetails",
-      route: urls.machines.machine.testing.scriptResult(null),
       path: urls.machines.machine.testing.scriptResult({
         id: "abc123",
         scriptResultId: 1,
       }),
     },
     {
       component: "NodeLogs",
-      route: urls.machines.machine.logs.index(null),
       path: urls.machines.machine.logs.index({ id: "abc123" }),
     },
     {
       component: "MachineConfiguration",
-      route: urls.machines.machine.configuration(null),
       path: urls.machines.machine.configuration({ id: "abc123" }),
     },
-    {
-      // Redirects to summary:
-      component: "MachineSummary",
-      route: urls.machines.machine.index(null),
-      path: urls.machines.machine.index({ id: "abc123" }),
-    },
-  ].forEach(({ component, path, route }) => {
-    it(`Displays: ${component} at: ${path}`, () => {
+  ].forEach(({ component, path }) => {
+    it(`Displays: ${component} at: ${path}`, async () => {
       const store = mockStore(state);
       const wrapper = mount(
         <Provider store={store}>
           <MemoryRouter initialEntries={[{ pathname: path }]}>
             <CompatRouter>
               <Routes>
-                <Route element={<MachineDetails />} path={route || "*/:id/*"} />
+                <Route element={<MachineDetails />} path={"/machine/:id/*"} />
```

Comment:
This might be better as `urls.machines.machine.index(null)` (as well as the other instances of `/machine/:id` in this file).